window/src/lib.rs

```diff
@@ -249,6 +249,14 @@ pub trait WindowOps {
     /// window movement on the server side (Wayland).
     fn request_drag_move(&self) {}
 
+    /// Signal to the windowing system that the mouse is over
+    /// a window dragging area.
+    ///
+    /// This is only implemented on backends that need to
+    /// know if the mouse is in a drag area to handle the
+    /// click before forwarding the event (Windows).
+    fn set_window_drag_position(&self, _coords: ScreenPoint) {}
```

Comment:
What do you think about making this accept a `Rect` that describes the title bar region as opposed to a single point?
I think having a single point might be prone to occasionally "missing" in the hit test if the cursor is moving rapidly.

Reply:
I think the point is better because it's precise, since the drag-able area of the tab bar is not a rectangle, it also includes the left padding for instance. I tried and couldn't cause the miss, seems like the event processing is fast enough, so I think the possibly very rare miss is acceptable.